hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/TestCleaner.java

```diff
@@ -1124,15 +1124,16 @@ public void testIncrementalFallbackToFullClean() throws Exception {
         put(p1, CollectionUtils.createImmutableList(file1P1, file2P1));
       }
     });
-    commitWithMdt("1", part1ToFileId, testTable, metadataWriter);
-    commitWithMdt("2", part1ToFileId, testTable, metadataWriter);
+    commitWithMdt("10", part1ToFileId, testTable, metadataWriter);
+    testTable.addClean("15");
+    commitWithMdt("20", part1ToFileId, testTable, metadataWriter);
 
     // add clean instant
     HoodieCleanerPlan cleanerPlan = new HoodieCleanerPlan(new HoodieActionInstant("", "", ""),
         "", "", new HashMap<>(), CleanPlanV2MigrationHandler.VERSION, new HashMap<>(), new ArrayList<>());
     HoodieCleanMetadata cleanMeta = new HoodieCleanMetadata("", 0L, 0,
-        "2", "", new HashMap<>(), CleanPlanV2MigrationHandler.VERSION, new HashMap<>());
-    testTable.addClean("3", cleanerPlan, cleanMeta);
+        "20", "", new HashMap<>(), CleanPlanV2MigrationHandler.VERSION, new HashMap<>());
```

Comment:
Any particular reason for changing all the instants to double digits?

Reply:
I add a clean instant between the original instant '1' and '2' to test that fallback logic is not influenced by clean. I could have add 1 to every instant after the clean instant I added, but make every instant 2 digit can leave some naming space between instants, incase someone want to add more instants in between in the future.

Reply:
My bad, noticed it now.